config/district_new_bedford.yml

```diff
@@ -82,3 +82,32 @@ schools:
   -
     name: Whaling City Jr./Sr. High
     local_id: '515'
+service_types:
```

Comment:
@alexsoble This is super interesting!  I'm excited to learn more about the adoption path for writing and reading this data in New Bedford.  Without knowing more about that, it's hard to comment more, but one thing that jumped out to me was that we're forking the way that we talk about these services and so they will stay pretty specific about what's happening.

Evaluating this more really depends on expectations and assumptions about the adoption path in New Bedford.  It might be good to capture more about that here if it's come up (or if not we'll maybe learn more shortly!)